docs/serving/samples/hello-world/helloworld-php/index.md

```diff
@@ -5,4 +5,210 @@ weight: 1
+```
+
+## Before you begin
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../../install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+
+## Recreating the sample code
+
+1. Create a new directory and cd into it:
+
+   ```shell
+   mkdir app
+   cd app
+   ```
+
+1. Create a file named `index.php` and copy the code block below into it:
+
+   ```php
+   <?php
+   $target = getenv('TARGET', true) ?: 'World';
+   echo sprintf("Hello %s!", $target);
+   ```
+
+1. Create a file named `Dockerfile` and copy the code block below into it. See
+   [official PHP docker image](https://hub.docker.com/_/php/) for more details.
+
+   ```docker
+   # Use the official PHP 7.3 image.
+   # https://hub.docker.com/_/php
+   FROM php:7.3-apache
+
+   # Copy local code to the container image.
+   COPY index.php /var/www/html/
+
+   # Use the PORT environment variable in Apache configuration files.
+   RUN sed -i 's/80/${PORT}/g' /etc/apache2/sites-available/000-default.conf /etc/apache2/ports.conf
+
+   # Configure PHP for development.
+   # Switch to the production php.ini for production operations.
+   # RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini"
+   # https://hub.docker.com/_/php#configuration
+   RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini"
+   ```
+
+1. Create a `.dockerignore` file to ensure that any files related to a local
+   build do not affect the container that you build for deployment.
+
+   ```ignore
+   Dockerfile
+   README.md
+   vendor
+   ```
+
+1. Create a new file, `service.yaml` and copy the following service definition
+   into the file. Make sure to replace `{username}` with your Docker Hub
+   username.
+
+   ```yaml
+   apiVersion: serving.knative.dev/v1
+   kind: Service
+   metadata:
+     name: helloworld-php
+     namespace: default
+   spec:
+     template:
+       spec:
+         containers:
+           - image: docker.io/{username}/helloworld-php
+             env:
+               - name: TARGET
+                 value: "PHP Sample v1"
+   ```
+
+## Building and deploying the sample
+
+Once you have recreated the sample code files (or used the files in the sample
+folder) you're ready to build and deploy the sample app.
+
+{{< tabs name="helloworld_ruby" default="kn" >}}
+{{% tab name="yaml" %}}
+
+1. Use Docker to build the sample code into a container. To build and push with
+   Docker Hub, run these commands replacing `{username}` with your Docker Hub
+   username:
+
+   ```shell
+   # Build the container on your local machine
+   docker build -t {username}/helloworld-php .
+
+   # Push the container to docker registry
+   docker push {username}/helloworld-php
+   ```
+
+1. After the build has completed and the container is pushed to docker hub, you
```

Comment:
```suggestion
1. After the build has completed and the container is pushed to Docker Hub, you
```

Capitalized for consistency. I think the base template being used for these examples has the same issue.